src/App/components/Resource/components/Examples/index.js

```diff
@@ -0,0 +1,190 @@
+import React, {Component} from 'react'
+import {map, includes, size, uniqueId} from 'lodash'
+import {
+  smallContainerWidth,
+  mediumContainerWidth,
+  largeContainerWidth,
+  xlargeContainerWidth,
+  xxlargeContainerWidth,
+} from 'package/utils/hardCodedSizes'
+import Heading from 'package/components/Heading'
+import Button from 'package/components/Button'
+
+const containerWidthActions = [
+  {
+    label: 'Small',
+    containerWidth: smallContainerWidth,
+  },
+  {
+    label: 'Medium',
+    containerWidth: mediumContainerWidth,
+  },
+  {
+    label: 'Large',
+    containerWidth: largeContainerWidth,
+  },
+  {
+    label: 'XLarge',
+    containerWidth: xlargeContainerWidth,
+  },
+  {
+    label: 'XXLarge',
+    containerWidth: xxlargeContainerWidth,
+  },
+]
+
+const containerBackgroundActions = [
+  {
+    label: 'Dark',
+    containerBackground: 'dark',
+  },
+  {
+    label: 'Light',
+    containerBackground: 'light',
+  },
+]
+
+const containerBoxSizing = 8
+
+export const optOuts = [
+  'containerWidth',
+  'containerBackground',
+  'types',
+]
+
+class Examples extends Component {
+
+  state = {
+    containerWidth: smallContainerWidth,
+    containerBackground: 'dark',
+    examples: false,
+  }
+
+  componentWillMount = () => {
+    this.handleCreateExamples()
+  }
+
+  handleContainerWidthChange = (containerWidth) => {
+    this.setState({
+      containerWidth,
+    })
+  }
+
+  handleBackgroundChange = (containerBackground) => {
+    this.setState({
+      containerBackground,
+    })
+  }
+
+  handleCreateExamples = () => {
+    this.setState({
+      examples: this.props.createExamples(),
+      renderId: uniqueId(),
```

Comment:
This forces the examples to update and grab new random values from faker when you hit "Randomize"